### ISSUE TITLE
Using i18n properties in the uiveri5 test for Fiori application

### ISSUE THREAD

**Issue by vaibhavhirani** [SAP/ui5-uiveri5#119]
We are testing Fiori Application using uiveri5 tests. Currently we are able to test our application by hardcoding values directly in the test file but we wanted to test our application for different locale also. So how can we use i18n properties in our tests ?

**Comment by maximnaidenov** [comment]
Hi,
you can use the I18NText or the LabelFor matchers: https://github.com/SAP/ui5-uiveri5/blob/master/docs/usage/locators.md
BTW. All OPA matchers from: https://openui5nightly.hana.ondemand.com/#/api/sap.ui.test.matchers are supported in uiveri5. just the syntax is declarative.

**Comment by vaibhavhirani** [comment]
Hi @maximnaidenov , thanks for the information. I am using the I18NText like this - 
I18NText : { propertyName: "text", key: "organizationalInformation"},. Where "organizationalInformation" is the key in i18n file. It doesn't seem to find the i18n properties in my application.
What are the steps to achieve this ? How can I load the model so the test file can access i18n properties ?
Can you provide any sample ?
Thanks.

**Comment by maximnaidenov** [comment]
The test does not load the model, the app does this. Are those the correct values that you see with UI5Inspector ?

**Comment by vaibhavhirani** [comment]
Hi @maximnaidenov ,

I have this element -
	
	orgInfo: parentElements.consultantHeaders.content.element(
      by.control({
        controlType: "sap.m.Title",
        properties: {
           text: "Organisation Information"
         }
      })),

I was testing this particular element whether this is present on page or not, by hard coding the text value to "Organisation Information". But we wanted to 
use the i18n properties and get the value from the i18n file as per the locale. So I tried below code.  	  
	  
	orgInfo: parentElements.consultantHeaders.content.element(
      by.control({
        controlType: "sap.m.Title",
        I18NText : { propertyName: "text", key: "organizationalInformation"}
      })),

And it had thrown me errors mentioned below - 

No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#__title5' -  sap.ui.test.matchers.I18NText
No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#__title8' -  sap.ui.test.matchers.I18NText
No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#__title10' -  sap.ui.test.matchers.I18NText
No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#__title11' -  sap.ui.test.matchers.I18NText
No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#consultantProfileUi::ConsultantProfileObjectPage--fe::table::skills::LineItem-title' -  sap.ui.test.matchers.I18NText
No value for the key 'organizationalInformation' in the model 'i18n' of 'Element sap.m.Title#consultantProfileUi::ConsultantProfileObjectPage--fe::table::roles::LineItem-title' -  sap.ui.test.matchers.I18NText

I changed the value corresponding to the 'organizationalInformation' key in order to check whether it works or not ( changing "Organisation Information" to "Organisation Informatio")
But no changes in the behaviour, throws the same errors.
